### PR TITLE
Polyhedron demo: save memory by clearing the vectors after they are sent to OpenGL

### PR DESCRIPTION
**Pull request by maxGimeno**
This should lower the RAM consumption for big items.

### CODE REVIEW

File: Polyhedron/demo/Polyhedron/Scene_polyhedron_selection_item.h [hunk 1]
```diff
@@ -775,12 +781,14 @@ public Q_SLOTS:
...
-
-    mutable QOpenGLShaderProgram *program;
+    mutable std::vector<float> positions_facets;
+    mutable std::vector<float> normals;
+    mutable std::vector<float> positions_lines;
+    mutable std::vector<float> positions_points;
+  mutable int nb_facets;
+  mutable int nb_points;
+  mutable int nb_lines;
+  mutable QOpenGLShaderProgram *program;
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene_polyhedron_selection_item.h]: Indentation issue, here.

File: Polyhedron/demo/Polyhedron/Scene_polylines_item.h [hunk 2]
```diff
@@ -92,15 +92,17 @@ public Q_SLOTS:
...
+    mutable std::vector<float> positions_center;
+    mutable std::vector<float> normals_spheres;
+    mutable std::vector<float> color_spheres;
+    mutable std::vector<float> color_wire_spheres;
+    mutable int nb_spheres;
+    mutable int nb_wire;
+    mutable int nb_centers;
+    mutable int nb_lines;
   mutable QOpenGLShaderProgram *program;
   mutable   GLuint nbSpheres;
```
Comment (lrineau) [inline Polyhedron/demo/Polyhedron/Scene_polylines_item.h]: Indentation!

**Comment by lrineau** [comment]
@maxGimeno Please fix the two indentation issues that were pointed out:
  - https://github.com/CGAL/cgal/pull/309#discussion_r38519756
  - https://github.com/CGAL/cgal/pull/309#discussion_r38519730

and then push the new commit to the branch `CGAL_RAM_Eco_GF` on your fork `mine`.

**Comment by maxGimeno** [comment]
Done

**Comment by lrineau** [comment]
Thanks.

**Pull request merged**